src/garage/torch/algos/sac.py

```diff
@@ -377,7 +420,7 @@ def _log_statistics(self, policy_loss, qf1_loss, qf2_loss):
 
         """
         with torch.no_grad():
-            tabular.record('alpha', self.log_alpha.exp().item())
+            tabular.record('alpha', self.log_alpha.exp().mean().item())
         tabular.record('policy_loss', policy_loss.item())
         tabular.record('qf_loss/{}'.format('qf1_loss'), float(qf1_loss))
```

Comment:
please make these tabular logs consistent with the rest of the codebase, i.e. use CamelCase and scope the various statistics within their respective primitives QFunction/Loss, etc.